packages/paste-website/src/pages/experiences/flex/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Flex
+description: An overview of the Flex experience's design guidelines.
+slug: /experiences/flex/
+---
+
+import {graphql} from 'gatsby';
+import {Card} from '@twilio-paste/card';
```

Comment:
Thought: I don't think all these components need to be imported because they're not used.

Reply:
Oh great catch! This is where I originally built the WatchThisSpace component so it's a remnant of that. Thanks Lee!